### ISSUE TITLE
Timeline images alt text missing

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#164]
http://test-dukes.pantheonsite.io/stories/history/

Images aren't pulling/displaying their alt text.

**Comment by caitlynmayers** [comment]
Fix verified.
